src/cltool.cpp

```diff
@@ -294,6 +294,12 @@ bool cltool_parseCommandLine(int argc, char* argv[])
 			g_commandLineOptions.replayDataLog = true;
 			g_commandLineOptions.replaySpeed = (float)atof(&a[4]);
 		}
+        else if (startsWith(a, "-resetEvb") || 
```

Comment:
I wonder if it's worth adding a parameter to startsWith to allow case insensitive matching.  This won't catch reseteVB or resetevB